go/logic/applier.go

```diff
@@ -378,7 +384,9 @@ func (this *Applier) WriteChangelog(hint, value string) (string, error) {
 }
 
 func (this *Applier) WriteAndLogChangelog(hint, value string) (string, error) {
```

Comment:
It looks like we never actually use the `string` return value from the `WriteAndLogChangelog`, `WriteChangelogState`, and `WriteChangelog` functions - what do you think about getting rid of it and just returning `error`?